chapter3.Rmd

```diff
@@ -209,224 +238,235 @@ test_object("titles", undefined_msg = msg, incorrect_msg = msg)
 test_object("star_wars_matrix", eq_condition = "equal",
             incorrect_msg = "Did you set the row and column names of `star_wars_matrix` correctly? Use `colnames(star_wars_matrix) <- region` for the column names; do a similar thing to name the rows.")
 test_output_contains("star_wars_matrix", incorrect_msg = "Don't forget to print out `star_wars_matrix` after you've named the rows and columns.")
-success_msg("Great! You're on the way of becoming an R jedi! Continue to the next exercise.")
+success_msg("Laugh it up, fuzzball! You're on the way of becoming an R Jedi! Continue to the next exercise.")
```

Comment:
There are a limited number of famous Star Wars quotes that fitted in success messages. I think it will get a laugh, but it's your call on whether to keep it.